resources/party.ts

```diff
@@ -14,6 +15,27 @@ const emptyRoleToPartyNames = () => {
   };
 };
 
+export type BasePartyMemberParamObject = {
+  role?: Role;
+  job?: Job;
+  id?: string;
+  name: string;
+  nick: string;
```

Comment:
Most triggers currently use `shortName` which is why I included it here to begin with. It seems that `nick` is taking a dual role/default to `shortName` logic path here which may be counterintuitive since everything else here should be a concrete value.

It may make sense to include both here instead, and have the default be `nick` still, so that users have the option of still using `shortName` for specific triggers?

Reply:
I'm not sure what you mean by this, or by concrete value. `data.ShortName` the function is both "first name" and "nickname substitution". Users don't really know anything about "short name" as a function and I felt like "nick" better summed up what it did.

What would both options be? Would `shortName` just be `firstName`? Would `nick` be `firstName` if it didn't have a substitution? I'm not sure when you would ever want to pick between these.

Reply:
Gotcha.

I think we *could* leave the nick field blank if there's no nick and then also have the default field be the short name and it would have the same end result for a user who picks "nick". However, I don't think end users will notice this or be able to do anything with that change either?

Reply:
Yeah, I think this is fine as-is. As I said in my last comment, I think that keeping `nick` as `nickname ?? firstName` is more user-friendly here.

Maybe it also makes sense to add `firstName` as a separate field but I feel like the user should be able to just set someone's nickname to their first name if they wanted to anyways.